### PR TITLE
surge: add page

### PR DESCRIPTION
**Pull request by hill**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by hill** [comment]
@jimfilippou made those changes, thanks 😄

File: pages/common/surge.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Surge
+
+> Simple, single-command web publishing.
+
+- Upload a new site:
```
Comment (sbrl) [inline pages/common/surge.md]: Where does it upload to?

File: pages/common/surge.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+`surge {{ /path/to/my-project }}`
+
+- Deploy site to custom domain (note DNS records must be changed on domain):
```
Comment (sbrl) [inline pages/common/surge.md]: Perhaps `note that the DNS records must be changed on the domain` would read better?

**Comment by sbrl** [comment]
Thanks for reviewing this PR, @jimfilippou! Unfortunately it doesn't count as an 'approved review' because you're not a part of the tldr-pages org though. Still, thanks for the help 😺

File: pages/common/surge.md (hunk 1 above)
Comment (hill) [reply pages/common/surge.md]: It uploads to surge.sh servers - clarified that in the most recent change

File: pages/common/surge.md (hunk 2 above)
Comment (hill) [reply pages/common/surge.md]: Haha - updated the wording in the most recent change 😄

**Comment by agnivade** [comment]
@hill - I have made some changes which simplifies the page a bit. Let me know if those look okay to you.

@jimfilippou - Thanks for the review. But we use snake_case in our tokens. And there is no space within token braces `{{}}`. And path does not begin with a `/`. I have corrected those.

**Comment by sbrl** [comment]
Thanks, @hill! (and that's a short username you've got there!)

**Comment by hill** [comment]
Four letter usernames for the win! @sbrl

**Pull request merged**